### ISSUE TITLE
VLC

### ISSUE THREAD

**Issue by Scumm868** [ItIckeYd/VodRecovery#18]
After I use the program and recover a link for the vod, i place it in the VLC (open network stream) area and press play but sometimes it plays the vod and other times it doesn't. How do i fix this?

**Comment by ItIckeYd** [comment]
@Scumm868 did you check if segments were valid? Is the vod older then 60 days? Do other vods work?

**Comment by Scumm868** [comment]
@ItIckeYd yes all the vods were not older than 60 days and sometimes other vods playin in vlc while others dont't.

What m3u8 player chrome extension are you speaking about?

**Comment by ItIckeYd** [comment]
@Scumm868  Try https://chrome.google.com/webstore/detail/play-hls-m3u8/ckblfoghkjhaclegefojbgllenffajdc?hl=en 

this is what I use to play m3u8 links.

**Comment by Scumm868** [comment]
@ItIckeYd I just tried it but the video is just loading and not playing.

This is the link if you want to try to get it to play:  https://twitchtracker.com/scumtk/streams/45410887180

**Comment by ItIckeYd** [comment]
@Scumm868 Segments are invalid.. meaning the .TS files that make up a m3u8 file are not available which makes the vod not playable.

**Comment by Scumm868** [comment]
@ItIckeYd In the future how do I know if segments are invalid?

**Comment by ItIckeYd** [comment]
@Scumm868 If the vod contains muted segments the script will ask you if you want to check segment availability. For vods that don't contain muted segments if you try and play a vod with muted segments you will see the duration of the vod for a short period of time then you will see it go blank.. See photo below.

If you see the duration as those lines highlighted in yellow its likely that segments are invalid.

![screenshot](https://user-images.githubusercontent.com/86701308/168830515-b17b583a-5e83-47e7-94d6-628da7c59882.PNG)

**Comment by Scumm868** [comment]
@ItIckeYd Damn thx for the help... So there is no way to retrieve it 😭
